examples/debian_shared_dependencies/BUILD.bazel

```diff
@@ -92,7 +92,7 @@ oci_image(
     }),
 )
 
-oci_tarball(
+oci_load(
```

Comment:
nit: is this used anywhere?

Reply:
not by our tests no, but there for the completeness of the example